src/nvc.rs

```diff
@@ -480,7 +607,10 @@ impl Nvc {
                         OPCODE_SYSTEM_REGISTER_ID_PSW => self.reg_psw(),
                         OPCODE_SYSTEM_REGISTER_ID_CHCW => {
                             logln!("WARNING: stsr chcw not yet implemented");
```

Comment:
nit: Should probably write "not fully implemented" to indicate parts of this register are implemented